### ISSUE TITLE
Scaffold Gemfile is inconsistent

### ISSUE THREAD

**Issue by krukow** [calabash/calabash#206]
Run `calabash generate` and then try and run `bundle install` in the directory generated.

```
 bundle install
Fetching gem metadata from https://rubygems.org/...........
Fetching version metadata from https://rubygems.org/..
Resolving dependencies......
Bundler could not find compatible versions for gem "cucumber":
  In Gemfile:
    calabash (>= 0) ruby depends on
      calabash-android (~> 0.4.9) ruby depends on
        slowhandcuke (>= 0) ruby depends on
          cucumber (>= 0) ruby

    calabash (>= 0) ruby depends on
      calabash-android (~> 0.4.9) ruby depends on
        slowhandcuke (>= 0) ruby depends on
          cucumber (>= 0) ruby

    calabash (>= 0) ruby depends on
      calabash-cucumber (~> 0.9.152) ruby depends on
        cucumber (~> 1.3.0) ruby

    cucumber (= 2.0) ruby
Bundler could not find compatible versions for gem "rubyzip":
  In Gemfile:
    calabash (>= 0) ruby depends on
      calabash-android (~> 0.4.9) ruby depends on
        rubyzip (~> 0.9.9) ruby

    calabash (>= 0) ruby depends on
      xamarin-test-cloud (~> 0.9.23) ruby depends on
        rubyzip (~> 1.1) ruby
```

**Comment by jmoody** [comment]
@krukow Reproduced.

**Comment by jmoody** [comment]
There are several problems:

1. Calabash version 1.2.3 depends on calabash-android, calabash-cucumber, etc.
2. The absolute version requirement for cucumber is incorrect.  it should be unversioned or '~> 2.0'.

**Comment by TobiasRoikjer** [comment]
@krukow @jmoody I think it's because it is a pre-release

**Comment by krukow** [comment]
@jmoody @TobiasRoikjer can we fix this -- why would Calabash depend on Calabash iOS/Android?

**Comment by TobiasRoikjer** [comment]
@krukow Calabash does not depend on them.

**Comment by jmoody** [comment]
@krukow 

The most recent, non-pre-release version of the calabash gem is 1.2.1.

When your Gemfile looks like this:

```
gem 'calabash'
```

You'll get version 1.2.1 - which was released on Sept 15 _2013_, back when 'calabash' was going to be a bridge gem.

This is fixed by #218

**Comment by jmoody** [comment]
Fix is merged @krukow
